randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -955,6 +955,11 @@ def get_random_size(minimum, maximum):
         else:
             boss_sizes = {}
 
+        if self.configuration.spring_ball:
+            spring_ball_item = "Morph Ball Bomb"
+        else:
+            spring_ball_item = "Spring Ball"
+
```

Comment:
1. Spring ball is a custom/independant item that can be collected now, yes?
2. If the item is set to bombs, does it mean that collecting bombs automatically grants you the ability to use spring, or do you still need the custom spring item in addition?

Reply:
Read this code as:

> By default, you must collect the `Spring Ball` item in order to use Spring Ball, unless the user has enabled the Spring Ball option, in which case Spring Ball functionality will be granted upon acquisition of `Morph Ball Bomb`

Reply:
Gotcha. My main confusion was from the doc where it wasn't quite clear to me whether it meant "Being able to jump in ball means collecting spring ball and morph ball bombs" or "once you collected morph ball bombs, you get the ability to jump in ball"